CodePush.ios.js

```diff
@@ -126,72 +126,106 @@ function checkForUpdate() {
  * releases, and displaying a standard confirmation UI to the end-user
  * when an update is available.
  */
-function sync(options = {}) {  
+function sync(options = {}, onSyncStatusChange, onDownloadProgress) {  
```

Comment:
Since these are callbacks not events, do we want to drop the "on" prefix and add the "Callback" suffix (e.g. syncStatusChangeCallback, downloadProgressCallback).


If you agree, we shou do also update the readme as well.